tests/test_dcm2bids.py

```diff
@@ -71,3 +71,62 @@ def test_dcm2bids():
 
     if os.name != 'nt':
         bidsDir.cleanup()
+
+def test_caseSensitive_false():
+    # Validate caseSensitive false
+    bidsDir = TemporaryDirectory()
+
+    tmpSubDir = os.path.join(bidsDir.name, DEFAULT.tmpDirName, "sub-01")
+    shutil.copytree(os.path.join(TEST_DATA_DIR, "sidecars"), tmpSubDir)
+
+    app = Dcm2bids(
+        [TEST_DATA_DIR],
+        "01",
+        os.path.join(TEST_DATA_DIR,
+                     "config_test_not_case_sensitive_option.json"),
+                     bidsDir.name)
```

Comment:
I think this part is indented wrong? 
```suggestion
        bidsDir.name)
```